uber/configspec.ini

```diff
@@ -377,6 +361,10 @@ __many__ = integer
 __many__ = integer
 
 
+[table_prices]
```

Comment:
What would be an example of adding table prices here that aren't default? Would it be something like:
```
[table_prices]
0.5 = 100
1 = 200
```?